portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/administration/auditing/Audit.testcase

```diff
@@ -1597,8 +1597,8 @@ definition {
 	@priority = 5
 	test ViewLoginFailureMessageWhenLDAPUserEnterWrongPassword {
 		property apacheds.enabled = "true";
-		property osgi.module.configuration.file.names = "com.liferay.portal.security.audit.wiring.internal.configuration.AuditLogContextConfiguration.config:com.liferay.portal.security.audit.router.configuration.LoggingAuditMessageProcessorConfiguration.config";
-		property osgi.module.configurations = "enabled=\"true\"";
+		property osgi.module.configuration.file.names = "com.liferay.portal.security.audit.wiring.internal.configuration.AuditLogContextConfiguration.config:com.liferay.portal.security.audit.router.configuration.PersistentAuditMessageProcessorConfiguration.config:com.liferay.portal.security.audit.router.configuration.LoggingAuditMessageProcessorConfiguration.config";
```

Comment:
@manuelecastro Hi Manuel, does this me all 3 configurations have the same file content?

Since you are added `bufferSize=0` to the content, you might consider separating out the content like this: https://github.com/liferay-appsec/liferay-portal/pull/1920/commits/6423020eae4fc7bf25f457d6cb6c15092673f1fb#diff-f0ab3c960f2f43191c71f66594f9caedd18c055cd2f03d57043b131e81e105b1R278-R279

Reply:
Also, if possible, verify the configuration output by test running some of these functional tests locally and go to `./bundle/osgi/configs` to see if corresponding configs generated by these properties are correct